docs/source/packs.rst

```diff
@@ -6,114 +6,183 @@ What is a Pack?
+    # Show an index entry for the pack
+    st2 pack show sensu
 
-Pack management is done by |st2| actions from `packs` pack, pun intended. Run ``st2 action list --pack packs`` for a list of pack management actions.
+Installing a Pack
+-----------------
 
-Some packs can be installed and run "as is" without any configurations.
+Installing a pack is simple:
 
 .. code-block:: bash
 
-    st2 run packs.install packs=docker,sensu repo_url=https://github.com/StackStorm/st2contrib.git
+    st2 pack install sensu
 
-This downloads the Sensu and Docker packs from the `StackStorm/st2contrib community repo on GitHub <https://github.com/StackStorm/st2contrib>`__, places them as local content under ``/opt/stackstorm/packs``, registers with |st2| and loads the content.
+    # You can also install multiple packs:
+    st2 pack install datadog github
 
-By default packs are installed from the |st2| community repo. Use ``repo_url`` parameter to install a pack from a fork of `st2contrib  <https://github.com/StackStorm/st2contrib>`__, or from a custom repo. The following example installs all the packs from `StackStorm/st2incubator <https://github.com/StackStorm/st2incubator>`__ - the repo where you find our experiments and work-in-progress:
+This command will download packs from the `StackStorm Exchange organization on GitHub <https://github.com/StackStorm-Exchange>`__, place them under ``/opt/stackstorm/packs``, and register them with |st2|.
+
+Essentially, ``st2 pack install`` works with git repositories: there is one for every pack in the Exchange, and you can install your own packs from git just as easily.
 
 .. code-block:: bash
 
-    st2 run packs.install packs=* register=all repo_url=https://github.com/StackStorm/st2incubator.git
+    st2 pack install https://github.com/emedvedev/chatops-training
 
-If you are using a private repo to host your pack(s) on github, use:
-``repo_url=https://username:password@github.com/username/repository.git``.
-You can optionally pass ``subtree=True`` if the repo contains multiple packs within ``packs`` directory
-of repo root just like `StackStorm/st2contrib community repo on GitHub <https://github.com/StackStorm/st2contrib>`_.
-If the repo contains only one pack at the repo root, then use ``subtree=False``.
+By default, the latest version of a pack will be installed, but you can specify a particular version, branch, tag, or even a commit hash.
 
-To uninstall packs: ``st2 run packs.uninstall packs=docker,sensu``. This unloads and unregisters the content and deletes the packs from the disk.
+.. code-block:: bash
 
-The integration packs often require configurations to adjust to the environment. e.g. you will need to specify SMTP server for email, a puppet master URL for Puppet, or a Keystone endpoint and tenant credentials for OpenStack. The installation process is:
+    st2 pack install cloudflare=77ee04e
+    st2 pack install cloudflare=0.1.0
+    st2 pack install https://github.com/emedvedev/chatops-training=testing
```

Comment:
Mmm. Why exactly? `https://github.com/emedvedev/chatops-training=training` is not a url, it's a `[pack]=[version]` with pack name as url and tag\branch as version. `[pack]#[version]` might have worked as url in some scenarios, but the team decided `#` is not a good delimiter for that purpose.